packages/graph-explorer/src/modules/ConnectionDetail/ConnectionDetail.tsx

```diff
@@ -38,7 +38,7 @@ export type ConnectionDetailProps = {
 
 const HEADER_ACTIONS = (isSync: boolean, isFileBased: boolean) => [
   {
-    label: "Synchronize Database",
+    label: "Sync Connection",
```

Comment:
Apologies for the confusion with my screenshot but I think we should keep the old label. Thanks!

Reply:
Hey, @agutierrezgit Updates have been pushed up.